components/KiboShipping.vue

```diff
@@ -49,11 +82,36 @@ export default {
       type: Array,
       default: () => [],
     },
+    isUserLoggedIn: {
+      type: Boolean,
+      default: false,
+    },
+    loadingSavedUserAddress: {
+      type: Boolean,
+      default: false,
+    },
   },
   setup(props, context) {
     const isReadonly = true
     const isValidShippingDetails = ref(false)
     const shippingAddress = ref({ ...props.value })
+    const isAddressFormOpen = ref(false)
+    const isSaveShippingAddresChecked = ref(false)
```

Comment:
Is this intentionally used in few places  'Addres' and for others 'Address' ?

Reply:
Resolved